components/usage/pkg/controller/reconciler.go

```diff
@@ -205,50 +241,3 @@ func trimStartStopTime(instances []db.WorkspaceInstanceForUsage, maximumStart, m
 	}
 	return updated
 }
-
-func groupInstancesByAttributionID(instances []db.WorkspaceInstanceForUsage) UsageReport {
```

Comment:
Instead of grouping here, and then un-grouping to insert the WorkspaceInstanceUsage records into the DB, we just not group. 

The grouping is then done in the BillingController as part of GetCreditReport()